plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/StoreConnect.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Connect store to Woo.com Task
+ */
+class StoreConnect extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'connect-store';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Connect store to Woo.com', 'woocommerce' );
+	}
+
+	/**
+	 * Content.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		return '';
+	}
+
+	/**
+	 * Time.
+	 *
+	 * @return string
+	 */
+	public function get_time() {
+		return __( '2 minutes', 'woocommerce' );
```

Comment:
Are we able to remove the time? We've transitioned away from displaying it next to the tasks as it's often misleading.

Reply:
Addressed 4b67662c32